src/statick_md/plugins/tool/writegood_tool_plugin.py

```diff
@@ -0,0 +1,91 @@
+"""Apply writegood tool and gather results.
+
+Website: https://github.com/btford/write-good
+"""
+
+import logging
+import re
+import subprocess
+from typing import List, Match, Optional, Pattern
+
+from statick_tool.issue import Issue
+from statick_tool.package import Package
+from statick_tool.tool_plugin import ToolPlugin
+
+
+class WriteGoodToolPlugin(ToolPlugin):  # type: ignore
+    """Apply writegood tool and gather results."""
+
+    def get_name(self) -> str:
+        """Get name of tool."""
+        return "writegood"
+
+    def scan(self, package: Package, level: str) -> Optional[List[Issue]]:
+        """Run tool and gather output."""
+        tool_bin = "writegood"
+
+        flags: List[str] = ["--parse"]
+        user_flags = self.get_user_flags(level)
+        flags += user_flags
+
+        files: List[str] = []
+        if "md_src" in package:
+            files += package["md_src"]
```

Comment:
do we want to run this on `rst_src` files as well?
https://github.com/sscpac/statick-md/blob/main/src/statick_md/plugins/discovery/rst_discovery_plugin.py#L43